python-sdk/tests/benchmark/datasets.md

```diff
@@ -41,6 +41,98 @@ Within `s3://astro-sdk-test/benchmark/fake_data/`, following are the files with
 | ten_gb     | 10 GB  | csv     | 385,007,364 | 7       | 10_gb.csv                 |
 
 
+## Origin of the synthetic datasets
+
+To make the benchmarking results uniform, we have generated synthetic dataset for various size for all the file types:
+
+1. CSV files
+
+- GCS path
+
+|            | size   | format | path to csv file(s)                                           |
+|------------|--------|--------|---------------------------------------------------------------|
+| ten_kb     | 10 KB  | csv    | gs://astro-sdk/benchmark/synthetic-dataset/csv/ten_kb.csv     |
+| hundred_kb | 100 KB | csv    | gs://astro-sdk/benchmark/synthetic-dataset/csv/hundred_kb.csv |
+| ten_mb     | 10 MB  | csv    | gs://astro-sdk/benchmark/synthetic-dataset/csv/ten_mb.csv     |
+| hundred_mb | 100 MB | csv    | gs://astro-sdk/benchmark/synthetic-dataset/csv/hundred_mb.csv |
+| one_gb     | 1 GB   | csv    | gs://astro-sdk/benchmark/synthetic-dataset/csv/one_gb.csv     |
+| two_gb     | 2 GB   | csv    | gs://astro-sdk/benchmark/synthetic-dataset/csv/two_gb.csv     |
+| five_gb    | 5 GB   | csv    | gs://astro-sdk/benchmark/synthetic-dataset/csv/five_gb.csv    |
+| ten_gb     | 10 GB  | csv    | gs://astro-sdk/benchmark/synthetic-dataset/csv/ten_gb.csv     |
+
+- AWS S3 path
+
+|            | size   | format | path to csv file(s)                                           |
+|------------|--------|--------|---------------------------------------------------------------|
+| ten_kb     | 10 KB  | csv    | s3://astro-sdk/benchmark/synthetic-dataset/csv/ten_kb.csv     |
+| hundred_kb | 100 KB | csv    | s3://astro-sdk/benchmark/synthetic-dataset/csv/hundred_kb.csv |
+| ten_mb     | 10 MB  | csv    | s3://astro-sdk/benchmark/synthetic-dataset/csv/ten_mb.csv     |
+| hundred_mb | 100 MB | csv    | s3://astro-sdk/benchmark/synthetic-dataset/csv/hundred_mb.csv |
+| one_gb     | 1 GB   | csv    | s3://astro-sdk/benchmark/synthetic-dataset/csv/one_gb.csv     |
+| two_gb     | 2 GB   | csv    | s3://astro-sdk/benchmark/synthetic-dataset/csv/two_gb.csv     |
+| five_gb    | 5 GB   | csv    | s3://astro-sdk/benchmark/synthetic-dataset/csv/five_gb.csv    |
```

Comment:
@sunank200 Are we adding multiple files for 5GB and 10GB files for CSV?

Reply:
Maybe we can just copy 10 * 100MB files in a folder?

Reply:
I thought these are single files?! Multiple files should be tested separately or?

Reply:
For csv I have created a single large 10 GB and 5 GB files but for ndjson I have created multiple files (folder)

Reply:
Okay, that's fine but we should test both cases - single file vs multi file or? If our sdk does not support 5gb file we should document it, if we have not already.

Reply:
Added both single file and multi-file in this https://github.com/astronomer/astro-sdk/pull/879/commits/6d437918e0027f45527a3a01b9833c82bedd0901.

We do support 5 GB file in loadfile

Reply:
I think we should create multiple files as well.